test/unit/govuk_index/elasticsearch_presenter_test.rb

```diff
@@ -10,29 +10,8 @@ def setup
   end
 
   def test_converts_message_payload_to_elasticsearch_format
-    payload = {
-      "base_path" => "/some/path",
-      "content_id" => "XXX-XXX-XXX-XXX",
-      "content_store_document_type" => "help_page",
-      "description" => "This page will help you love cheese too",
-      "details" => { "body" => "We love cheese" },
-      "document_type" => "help_page",
-      "expanded_links" => {
-        "organisations" => [
-          {
-            "content_id" => "YYY-YYY-YYY",
-            "title" => "The Great Cheese Organisation",
-          }
-        ]
-      },
-      "payload_version" => 1,
-      "public_updated_at" => "2016-02-29T09:24:10Z",
-      "publishing_app" => "rails_for_the_win",
-      "rendering_app" => "react_rules_ok",
-      "title" => "This is a help page",
-    }
-
-    presenter = elasticsearch_presenter(payload, "help_page")
+    payload = File.read(File.join(__dir__, 'fixtures/publishing_api_payload.json'))
```

Comment:
We should definitely validate against the notification schema here as it could easily become invalid over time.

I think my preference would be to split to up. We could use the schema helpers gem to and then override a specific set of fields we want to test.